### DISCUSSION TITLE
Defining conversion rules from the python side?

### DISCUSSION THREAD

**Discussion by jClugstor** [cjdoris/PythonCall.jl#353] [Q&A] [answered]
Is there a way to define conversion rules from python. Right now I have these conversion rules made in Julia, and I'm just using `seval` to register them. But I was wondering if there was a better way to do it so that I can use the conversion rules in a python project.

**Comment by cjdoris** [comment, marked as answer]
There's no official route from the Python side - using `seval` is probably the simplest.
